README.md

```diff
@@ -17,6 +17,12 @@ $ source /opt/netbox/venv/bin/activate
 (venv) $ pip3 install .
 ```
 
+To ensure NetBox Onboarding plugin is automatically re-installed during future upgrades, create a file named `local_requirements.txt` (if not already existing) in the NetBox root directory (alongside `requirements.txt`) and list the `nextbox-ui-plugin` package:
```

Comment:
Hi Ryan. Thank you for this valuable note. Please fix a typo in a plugin name (it is NextBox UI, not NetBox Onboarding).

Reply:
Ha my copy and paste from my PR for NetBox Onboarding plugin.  Fixed.